### DISCUSSION TITLE
נגזרת מכוונת או מלווה תורת הזרימה FLD1_004

### DISCUSSION THREAD

**Discussion by talkazlov** [NuclearGandhi/technion_second_brain#13] [טעויות ותיקונים]
במשוואת קושי פעם אחת כתוב נגזרת מלווה ובסוף הפיתוח כתוב נגזרת מכוונת
תודה לצוות הפאנגיה
![Screenshot 2024-08-26 133638](https://github.com/user-attachments/assets/575dd0e8-ab70-4b20-9eda-b3251761b17c)
![Screenshot 2024-08-26 133653](https://github.com/user-attachments/assets/ee16e095-81dc-4e83-ade0-d6c1c3b9bdd3)

**Comment by NuclearGandhi** [comment]
מלך, תודה טל
